### PR TITLE
Added Weather App Project

### LINKED ISSUE #204: Add Weather App to Projects
## 🚀 Feature

To add a new project called "Weather App" in [Basic_Web_Development](https://github.com/Kushal997-das/Project-Guidance/tree/main/Web%20Development/Basic). It will use basic Vanilla HTML CSS and Javascript.

we will learnm how to Implement: 
* use of API , Fetch etc

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

- [x] yes, read  [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md)

## Motivation

A weather app is one of the basic projects a web developer should learn how to make. It teaches a lot of essential topics in js. For me, I'm excited to share my knowledge and help people set foot on the right stones through simple and well- structured projects for everyone to benefit from. I will make it a tutorial style program with lots of space for Customization.

## Pitch

* it uses vanilla js, html and css
* basic use of API, api Keys and Fetch
* runs on browser like a website/web app

screenshot of project 

Screenshot (9)

### PR DESCRIPTION
**Pull request by AdmiralAnne**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

- [x]  YES

### Description

Added a new project called "weather app" to BasicWebDevelopment - It runs on the browser and doesn't need any frameworks. Runs using simple html, css and JavaScript.

Fixes #204 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

none

### CODE REVIEW

File: Web Development/Basic/Weather App/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+___
+### Weather App - JS, HTML, CSS
+A simple weather app created using HTML, CSS and JavaScript. 
+you can View a demo [here](https://admiralanne-js-weather.netlify.app/).
+>
+___
+### Screenshot ~
+>Insert City Name in search Box ~ 
+
+![alt text for screen readers](Icons\screenshot.png "Text to show on mouseover")
```
Comment (Kushal997-das) [inline Web Development/Basic/Weather App/README.md:10]: This link is not working please fix this @AdmiralAnne

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Please fix this bug @AdmiralAnne

File: Web Development/Basic/Weather App/README.md (hunk 1 above)
Comment (prakhar3062) [reply Web Development/Basic/Weather App/README.md:10]: Link is working but may be there is issue in image 
Screenshot (626)

**Review by prakhar3062 — APPROVED** [review]
@photon149 okay from my side

**Review by photon149 — APPROVED** [review]
Looks good to me 
@Kushal997-das please review and merge the same

**Review by Kushal997-das — APPROVED** [review]
@AdmiralAnne Good 👍

**Pull request merged**